components/app/landing/LandingDaoProposals.vue

```diff
@@ -3,30 +3,30 @@
     <h2>{{ $t('landing.dao_proposals.title') }} <span class="text-primary"> {{ $t('landing.dao_proposals.titleHighlight') }}</span></h2>
     <p>{{ $t('landing.dao_proposals.text') }}</p>
     <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-2 xl:grid-cols-4 2xl:grid-cols-4 gap-4 mt-10">
-      <div v-for="project in projects" :key="project.title">
+      <div v-for="daoProposal in daoProposals" :key="daoProposal._id">
```

Comment:
question (blocking): What kind of `_id` is this? I hope we do not share any document IDs from our mongoDB here?